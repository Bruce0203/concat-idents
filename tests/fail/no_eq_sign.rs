concat_idents::concat_idents!(fn_name no_eq {
    fn fn_name() {
        /* there must be an eq sign */
    }
});

fn main() {}
