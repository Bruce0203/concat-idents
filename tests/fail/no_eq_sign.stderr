error: expected `=`
 --> $DIR/no_eq_sign.rs:1:39
  |
1 | concat_idents::concat_idents!(fn_name no_eq {
  |                                       ^^^^^
